Makefile

```diff
@@ -32,16 +32,19 @@ ANSIBLE_BASE_IMAGE = quay.io/operator-framework/ansible-operator
 HELM_BASE_IMAGE = quay.io/operator-framework/helm-operator
 SCORECARD_PROXY_BASE_IMAGE = quay.io/operator-framework/scorecard-proxy
 SCORECARD_TEST_BASE_IMAGE = quay.io/operator-framework/scorecard-test
+SCORECARD_TEST_KUTTL_BASE_IMAGE = quay.io/operator-framework/scorecard-test-kuttl
```

Comment:
We've got to start consolidating these image builds. We have a _ton_ of deploy jobs on merge with master. For a follow-up.

Reply:
IMO we need have the targets to run/exec each one if we just want a specific image. However, I agree that the setup in the Travis master now has a lot of images and we could track a task to try aggregate then if it will reduce the time.